examples/region_sharding/101_initial_cluster.sh

```diff
@@ -30,8 +30,10 @@ fi
 CELL=zone1 ../common/scripts/vtctld-up.sh
 
 # start unsharded keyspace and tablet
-CELL=zone1 TABLET_UID=100 ../common/scripts/mysqlctl-up.sh
-SHARD=0 CELL=zone1 KEYSPACE=main TABLET_UID=100 ../common/scripts/vttablet-up.sh
+for T_UID in 100 101; do
```

Comment:
We need two tablets here because the `vtgate` start-up script waits for `primary,replica` and with a single tablet we end up with no replicas.

Reply:
Can we change vtgate to only wait for PRIMARY?

Reply:
The script that starts the `vtgate` (`../common/vtgate-up.sh`) is common for all tests, not only `region_sharding`, so changing it would affect all tests. It seemed less intrusive of a change to only have this test changed by having an extra `vttablet`. But if you prefer, I can do it that way. LMK.

Reply:
OK, that's fine. If people start running out of resources while running the example, we can revisit.